code/game/objects/items/stacks/tiles/tile_types.dm

```diff
@@ -117,6 +117,44 @@
 	stacktype = /obj/item/stack/tile/floor
 	build_type = /obj/item/stack/tile/floor
 
+var/list/cyborg_floor = list(
+	"steel techfloor" = /obj/item/stack/tile/floor/steel/techfloor,
+	"gray platform" =  /obj/item/stack/tile/floor/steel/gray_platform,
+	"cafe floor tile" = /obj/item/stack/tile/floor/cafe,
+	"maint floor tile" = /obj/item/stack/tile/floor/techmaint,
+	"perforated maint floor tile" = /obj/item/stack/tile/floor/techmaint/perforated,
+	"panel maint floor tile" = /obj/item/stack/tile/floor/techmaint/panels,
+	"cargo maint floor tile" = /obj/item/stack/tile/floor/techmaint/cargo,
+	"steel techfloor tile with vents" = /obj/item/stack/tile/floor/steel/techfloor_grid,
+	"steel brown perforated tile" = /obj/item/stack/tile/floor/steel/brown_perforated,
+	"steel gray perforated tile" = /obj/item/stack/tile/floor/steel/gray_perforated,
+	"steel cargo tile" = /obj/item/stack/tile/floor/steel/cargo,
+	"steel bar flat tile" = /obj/item/stack/tile/floor/steel/bar_flat,
+	"steel bar dance tile" = /obj/item/stack/tile/floor/steel/bar_dance,
+	"steel bar light tile" = /obj/item/stack/tile/floor/steel/bar_light,
+	"white floor tile" = /obj/item/stack/tile/floor/white,
+	"white cargo tile" = /obj/item/stack/tile/floor/white/cargo,
+	"red carpet" = /obj/item/stack/tile/carpet,
+	"black carpet" = /obj/item/stack/tile/carpet/bcarpet,
+	"blue carpet" = /obj/item/stack/tile/carpet/blucarpet,
+	"turquoise carpet" = /obj/item/stack/tile/carpet/turcarpet,
+	"silver blue carpet" = /obj/item/stack/tile/carpet/sblucarpet,
+	"purple carpet" = /obj/item/stack/tile/carpet/purcarpet,
+	"orange carpet" = /obj/item/stack/tile/carpet/oracarpet
+)
```

Comment:
Isn't it better to makt it static internal variable for `/obj/item/stack/tile/floor/cyborg/`?

Reply:
https://github.com/discordia-space/CEV-Eris/pull/4226/commits/c1aa7470d950be03f67fda7f7e17f02eb2a4d162